src/panels/config/hardware/ha-config-hardware.ts

```diff
@@ -68,59 +81,47 @@ class HaConfigHardware extends LitElement {
               <div class="content">
                 <ha-card outlined>
                   <div class="card-content">
-                    <ha-settings-row>
-                      <span slot="heading"
-                        >${this.hass.localize(
-                          "ui.panel.config.hardware.board"
-                        )}</span
-                      >
-                      <div slot="description">
-                        <span class="value">${this._OSData.board}</span>
-                      </div>
-                    </ha-settings-row>
+                    ${this._OSData?.board
+                      ? html`
+                          <ha-settings-row>
```

Comment:
Do we need a settings row if it's just the title. Maybe it can be the header of the card ?

Reply:
If we dont include the board code.